elements/lisk-chain/src/slots.ts

```diff
@@ -13,40 +13,30 @@
  */
 
 interface SlotsInput {
-	readonly epochTime: string;
 	readonly interval: number;
 }
 
 const SEC_IN_MS = 1000;
 
 export class Slots {
-	private readonly _epochTime: Date;
 	private readonly _interval: number;
 
-	public constructor({ epochTime, interval }: SlotsInput) {
-		this._epochTime = new Date(epochTime);
+	public constructor({ interval }: SlotsInput) {
 		this._interval = interval;
 	}
 
-	public getEpochTime(): number {
-		return Math.floor((Date.now() - this._epochTime.getTime()) / SEC_IN_MS);
-	}
-
 	public blockTime(): number {
 		return this._interval;
 	}
 
+	// eslint-disable-next-line class-methods-use-this
 	public getRealTime(time: number): number {
```

Comment:
I think we can remove this function as it doesn't make sense anymore

Reply:
It's only used in one place so far but I thought of leaving it to avoid making a different calculation if needs to be used somewhere else in the future but ok to remove. So should I keep it with a better name in case is used somewhere else in the future or just remove directly and when and if the time comes we create a new method in the future? @shuse2

Reply:
I think since we use unixtime, and the time is just time in sec, we don't need to keep this function.
Also, the only usage in the forger.ts can be updated to just multiply by 1000.

Reply:
Yes I was just considering if we might need to use it somewhere else even although is trivial but will remove.